docs/en_US/Tutorial/HowToUseSharedStorage.rst

```diff
@@ -0,0 +1,49 @@
+**How to Use Shared Storage**
+=============================
+
+If you want to use your own storage during using NNI, shared storage can satisfy you.
+Instead of using training service native storage, shared storage can bring you more convenience.
+All the information generated by the experiment will be stored under ``/nni`` folder in your shared storage.
+All the output produced by the trial will be located under ``/nni/{EXPERIMENT_ID}/trials/{TRIAL_ID}/nnioutput`` folder in your shared storage.
+This saves you from finding for experiment-related information in various places.
+Remember that your trial working directory is ``/nni/{EXPERIMENT_ID}/trials/{TRIAL_ID}``, so if you upload your data in this shared storage, you can open it like a local file in your trial code without downloading it.
+And we will develop more practical features in the future based on shared storage.
+
+.. note::
+    Shared storage is currently in the experimental stage. We suggest use AzureBlob under Ubuntu/CentOS/RHEL, and NFS under Ubuntu/CentOS/RHEL/Fedora/Debian for remote.
+    And make sure your local machine can mount NFS or fuse AzureBlob and has ``sudo`` permission on your remote runtime. We only support shared storage under training service with reuse mode for now.
+
+Example
+-------
+If you want to use AzureBlob, add below to your config. Full config file see :githublink:`mnist-sharedstorage/config_azureblob.yml <examples/trials/mnist-sharedstorage/config_azureblob.yml>`.
+
+.. code-block:: yaml
+
+    sharedStorage:
```

Comment:
add doc for these new added configuration in ExperimentConfig.rst?

Reply:
add them